### DISCUSSION TITLE
关于npm install报错以及本地静态页面异常的问题

### DISCUSSION THREAD

**Discussion by gwyneth-zgf** [YunYouJun/hexo-theme-starry#102] [Q&A 疑难杂症] [answered]
按云游君的小白教程和CSDN上的教程操作过好几次了，试着在搜索引擎上找解答，but以下问题还是解决不了呜呜呜
![6}I(XSFHGH`COS4VY(RA5HC](https://user-images.githubusercontent.com/72310956/105739008-9d020d80-5f72-11eb-9b7b-c66300deaf4e.png)
![SFQV~DJC)QTF(@F29_ALVD3](https://user-images.githubusercontent.com/72310956/105739047-a8edcf80-5f72-11eb-8704-0bebe94fb572.png)
不知道有没有解决的办法呢？

**Comment by YunYouJun** [comment, marked as answer]
`node is not recognized` 这是 nodejs 没有全局安装，添加到环境变量吧。重新安装下看看。以及最好提供 Node 版本。

**Reply by gwyneth-zgf** [reply]
整了三天终于出来啦！！（谁能想到这种跟教程的东西弄了这么久呢 害）node版本都可以查看，根据不同博主的建议增删了n次环境变量，云里雾里地但总算弄出来了。谢谢谢谢yunyoujun，你的主题很好康：）

**Reply by YunYouJun** [reply]
Thanks. 祝用的开心。
